cmd/main.go

```diff
@@ -64,6 +81,18 @@ func main() {
 	gracefulShutdown(srv.HTTPServer(), srv.Config().GetGracefulTimeout())
 }
 
+func printConfig(cfg *configuration.Registry) {
```

Comment:
Let's maybe move this function it to `cfg *configuration.Registry` as a method?

Reply:
done as part of: 1863479426aebd4c11524c081a17b3773e64e32c